### DISCUSSION TITLE
Modelleerprincipes -  U1. Informatiemodel bevordert samenhang

### DISCUSSION THREAD

**Discussion by pmaria** [Geonovum/disgeo-imsor#52] [General] [answered]
Modelleerprincipe [[U1]](https://geonovum.github.io/disgeo-imsor/modelleerprincipes/#u1-informatiemodel-bevordert-samenhang) - Het informatiemodel (IMSOR) bevordert samenhang tussen informatieobjecten in registraties

Discussie uit expertgroepsessie:

> - Lennart: betekent dit als implicatie dat IMSOR alleen een conceptueel informatiemodel kan zijn?
> - Lennart: fundamentele vraag: de registraties hebben al informatiemodellen, dus het is raar om een informatiemodel te maken; eigenlijk wil je samenhang tussen die modellen; het lijkt hier [U1] te gaan om een semantisch model, niet een informatiemodel.
> - Jan: gegevens = data; informatie = bewerkte data. Er is een verschil tussen samenhang op het niveau van data en het niveau van informatie.
> - Niels: je moet dit [U1] lezen in licht van andere definities hierbinnen. Dan lijkt het hier te gaan om samenhang tussen de objecten. Het zegt nog niks over samenhang tussen bestaande IMs.
> - Michel: het is niet duidelijk of IMSOR huidige IMs gaat vervangen of dat het hierboven ligt. 
> - Paul: IMSOR lijkt te duiden op een topmodel.
> - Lennart: Als voorbeeld [om de positie van SOR beter te begrijpen]: als er een relatie is tussen een brug (BGT) en een perceel (BRK) moet je dat in de registraties kunnen registreren, of alleen op een conceptueel niveau?. Als dit tweede het geval is, dan is het afleidbaar (ie bewerkte data), en dit hoort niet thuis in een logisch model.
> - Niels: Heel simpel gezegd [over U1] willen we meer samenhang dan in de huidige registraties. Mensen hadden bij het opstellen van deze punten waarschijnlijk niet voor ogen hoe dit precies moest. Dit kunnen we nog op verschillende manieren realiseren. Zo moet je dit uitgangspunt interpreteren.

**Comment by lvdbrink** [comment]
> Lennart: fundamentele vraag: de registraties hebben al informatiemodellen, dus het is raar om een informatiemodel te maken; eigenlijk wil je samenhang tussen die modellen; het lijkt hier [U1] te gaan om een semantisch model, niet een informatiemodel.

Toen we dit uitgangspunt voor het eerst opschreven, was de gedachte nog dat we een nieuw IMSOR informatiemodel zouden maken, als "stip op de horizon", en dat de huidige registraties op termijn zouden kunnen fuseren tot één geobasisregistratie waar het IMSOR dan het informatiemodel van zou zijn. 

Maar inmiddels denken we meer in de richting van wat Lennart hierboven zegt. In ons PvA wordt nog wel gesproken van een "informatiemodel", maar de aanpak is om stapsgewijs de bestaande registraties steeds meer te integreren: 

> [DOEL:] Het via een stapsgewijze benadering doorontwikkelen richting een informatiemodel dat er voor zorg draagt dat de objecttypen in goede semantische samenhang zijn gebracht. Hiermee kan dit informatiemodel en de onderliggende basisregistratie(s) het fundament verder verstevigen voor overheidsinformatiehuishouding. 

We willen dit doen door 
- in 'spoor 1' zoveel mogelijk samenhang realiseren boven op de bestaande voorzieningen; met een over deze voorzieningen heen liggende integrale semantische laag; hier kunnen we ook gaan ervaren welke samenhang echt belangrijk is en moeilijk in een 'laagje eroverheen' te realiseren is. i.e. welke relaties asserted ipv derived zouden moeten zijn. 
- in 'spoor 2' is er reguliere doorontwikkeling binnen losse basisregistraties en de bijbehorende informatiemodellen 
- in 'spoor 3' harmoniseren we generieke onderwerpen - hier zijn we in de expertgroep in feite al mee bezig: de modelleerprincipes zijn hier onderdeel van maar ook het modelleren van historie, metadata, waardelijsten, etc. 
- in 'spoor 4' gaat het om het stapsgewijs harmoniseren & integreren van losse basisregistraties. Hiertoe worden uit het [EMSO](https://docs.geostandaarden.nl/disgeo/emso/) document eerst de relevante functionele eisen gehaald, als wijzigingsverzoek geformuleerd, impact bepaald en geprioriteerd. Vervolgens worden ze stuk voor stuk opgepakt in de registraties waar ze aan raken. 

"Het informatiemodel" in deze aanpak lijkt dus inderdaad meer een semantisch model te zijn. 

Wellicht kunnen we voor de generieke onderwerpen wel onderdelen in een informatiemodel modelleren zodat deze te hergebruiken zijn in de modellen van de registraties.

Ik ben heel benieuwd naar de mening van de experts hierover want het kan zeker nog scherper!

**Reply by RiX012** [reply]
Ik denk dat dit een goede aanpak is. Zeker als je bij het doorlopen van spoor 1 en 3:

1. De modelleerstandaarden leidend laat zijn
2. Je overwegingen en interpretaties goed vastlegt
3. Beargumenteerd waarom zaken gemaakt zijn
4. Het zo klein mogelijk houdt
5. Het zo generiek mogelijk houdt
Met name punt 5 vind ik van belang omdat op den duur dan meer (basis)registraties en informatiemodellen gecommiteert kunnen worden aan de aanpak en het semantisch model.

**Reply by pmaria** [reply]
> "Het informatiemodel" in deze aanpak lijkt dus inderdaad meer een semantisch model te zijn.

In termen van MIM gaan we in mijn optiek een model van begrippen, een conceptueel model en een logisch model maken. Of de SOR vervolgens geïmplementeerd wordt als een registratie conform dit datamodel, of als een interactielaag boven bestaande registraties met bestaande informatiemodellen is, dat is volgens mij een tweede.

Het uitgangspunt hier heeft als doel om er voor te zorgen dat het informatiemodel informatieobjecten oplevert die in "in samenhang" bruikbaar zijn. Bijvoorbeeld, door het relateren daaraan vanuit andere registraties, of door het aanvullen van informatie over SOR-objecten vanuit andere registraties.

**Comment by lvdbrink** [comment]
Discussie uit expertgroep 17/6: 

- Pano: in mijn optiek maken we een begrippenkader, conceptueel model en logisch model.
- Niels: eens.
- Pano: dan moeten we het nu nog verduidelijken. Wat er nu staat in 4.1 moet dus wel uitgebreid worden.
- Lennart: samenhang waartussen? Beter expliciet maken. Dit leest nu alsof je informatie naar die registratie gaat sturen, zo lees ik "koppelen". Maar met het idee ben ik het wel eens. Misschien "samen te stellen" gebruiken?  Of je kunt zeggen dat we hier de objecttypes beschrijven, en de onderliggende implicatie is dat we die gegevens moeten hebben als voorzieningen. Los van of het wel/niet registraties zijn.
- Linda: ik zit nog na te denken, hieruit [uit de Github discussie] volgt dat je een begrippenmodel/conceptueel model en logisch model nodig hebt. Kun je uitleggen waarom?
- Pano: het was eigenlijk een reactie op een comment van Lennart. Dit volgde meer uit de opmerkingen waarin een semantisch model genoemd werd. Ik probeerde in MIM termen te beschrijven wat we maken. 
- Lennart: ik hou er van als we semantische informatiemodellen maken - hoe meer je dit doet, hoe meer mensen het begrijpen. Maar we hebben als streven voor de SOR dat het samenhang brengt op het niveau van logische modellen. Als je je tot de huidige modellen richt, dan ben je beperkt in wat er nu staat en wat mogelijk is vanuit deze modellen. Het alternatief is dat je vertrekpunt een semantisch model is, en dan kijk je hoe je de huidige modellen hierop kunt afstemmen. 
- Linda: wij doen dus het eerste vooral: ons uitgangspunt zijn de logische modellen die al bestaan. En dan kijken wat er in die modellen te verbeteren valt zodat er meer samenhang komt. En dus niet: een prachtig informatiemodel  maken en dan hopen dat de registraties worden samengevoegd of zichzelf gaan aanpassen. Het informatiemodel zal een soort geharmoniseerd model worden waar registraties hun eigen model op kunnen mappen; en zoveel mogelijk of in ieder geval waar nodig en mogelijk passen we de bronregistraties aan. Natuurlijk is het iets genuanceerder, want de waarheid ligt ergens in het midden. 

Dus we willen niet een ideaalplaatje modelleren en ook niet "redden wat te redden valt", maar wel het maximale eruit halen. Het zou mogelijk zo kunnen gaan werken: "Voorziening ik wil jouw informatie niet zoals je die zelf hebt opgeslagen, ik wil jouw informatie volgens dit model (van de SOR)".

**Comment by lvdbrink** [comment, marked as answer]
Uitgangspunt 1 is goed. Bovenstaande discussie gaat eigenlijk over het hoe. Dit is zeker een belangrijk punt dat nog verder uitgewerkt moet worden.

Voorlopig gaan we ervan uit dat er een begrippenkader, conceptueel model en logisch model nodig zullen zijn. Deze drie dingen willen we in een high 5 eind augustus verder uitwerken.

**Comment by lvdbrink** [comment]
U1 is vastgesteld in [expertgroep 8 juli](https://github.com/Geonovum/disgeo-imsor/blob/master/overleg/08-07-2021-aantekeningen.md).
